src/main/java/org/embulk/util/aws/credentials/AwsCredentials.java

```diff
@@ -171,13 +198,42 @@ public void refresh() {
             };
         }
 
+        case "role":
+        {
+            reject(task.getAccessKeyId(), accessKeyIdOption);
+            reject(task.getSecretAccessKey(), secretAccessKeyOption);
+            reject(task.getSessionToken(), sessionTokenOption);
+            reject(task.getProfileFile(), profileFileOption);
+            reject(task.getProfileName(), profileNameOption);
+            String accountId = require(task.getAccountId(),
+                    "'" + accountIdOption + "'");
+            String roleName = require(task.getRoleName(),
+                    "'" + roleNameOption + "'");
+            String externalId = require(task.getExternalId(),
+                    "'" + externalIdOption + "'");
+
+            String arn = String.format(ARN_PATTERN, accountId, roleName);
+            String sessionName = UUID.randomUUID().toString();
```

Comment:
For session name, could we add another configuration like `treasure-data-UUID` or re-use the `prefix` is also okay. 
```
    @Override
    @Config("session_prefix")
    @ConfigDefault("\"treasure-data\"")
    String getSessionPrefix();
```

This would make the session name readable and could be specific rather than an UUID

Reply:
I think we should use session_name and let use configure a name which they want.